pkg/generic/thriftidl_provider.go

```diff
@@ -17,19 +17,28 @@
 package generic
 
 import (
+	"context"
 	"fmt"
+	"os"
 	"path/filepath"
 	"sync"
 
+	dthrift "github.com/cloudwego/dynamicgo/thrift"
 	"github.com/cloudwego/thriftgo/parser"
 
 	"github.com/cloudwego/kitex/pkg/generic/descriptor"
 	"github.com/cloudwego/kitex/pkg/generic/thrift"
 )
 
+const (
+	GoTagDisabled    = "KITEX_GENERIC_GOTAG_ALIAS_DISABLED"
```

Comment:
Repeated definition of KITEX_GENERIC_GOTAG_ALIAS_DISABLED, reuse it with isGoTagAliasDisabled

Reply:
fixed